types/ember-useragent/tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
Do you need this? (I understand `ember__component` is using this lib)

---

The `fastboot` service, which is referenced in the `user-agent` service, uses the `Headers` type. Removing "dom" causes an error when running `npm test`.